extensions/panache/hibernate-orm-panache-rest/deployment/src/main/java/io/quarkus/panache/rest/hibernate/orm/deployment/EntityDataAccessImplementor.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkus.panache.rest.hibernate.orm.deployment;
```

Comment:
The extension name is `quarkus-hibernate-orm-panache-rest`, which is consistent with `quarkus-hibernate-orm-panache`, but this package is upside-down. Shouldn't it be `io.quarkus.hibernate.orm.panache.rest.deplayment`, which would be consistent with `io.quarkus.hibernate.orm.panache.deployment`?

Reply:
Hmm good question. We should take into account the common and future mongo modules.
I think the panache prefix `io.quarkus.panache.rest.*` makes more sense to group them all together.  Maybe we should rename the module to `quarkus-panache-rest-hibernate-orm`?

Reply:
Well, the mongo module name would work using both name options. I tend to think that because it's tightly related to ORM or Mongo, it's a sub-module of them, so I'd favour `quarkus-hibernate-orm-panache-rest` over `quarkus-panache-rest-hibernate-orm`.

But perhaps @emmanuelbernard or @loicmathieu have opinions?

Reply:
Do we plan to have a common quide for both extension or separate guides ?

If we plan to have a common guide `quarkus-panache-rest-hibernate-orm` is more logical so we can name the guide "Automatic REST endpoint with Panache"  saying  that it comes in two implementations Hibernate and Mogodb in the two flavors of Panache repository and entity.

If we plan to have a guide for each implementation (or to integrate the documentation in the existing guides) so it's more logical to use `quarkus-panache-hibernate-orm-rest` as the REST entpoint will be a functionality of hibernate ORM Panache ...

Reply:
I prefer the common approach

Reply:
That's a good question. OTOH we currently have two guides mongo-panache and orm-panache.

Really not sure how to proceed. @emmanuelbernard ?

Reply:
I'd be inclined to not do common guide for Panache stuff.
Just create "Technology Foo" with Panache and its own guide.

But here what is the use case? REST endpoints form the database so we could have a guide explaining how to do it with ORM and Mongo etc (if there are specificities). Id id not check the PR sorry.

Reply:
I don't think he meant a common guide for the whole Panache, just for the REST extensions for Hibernate and Mongo.

This extension generates REST endpoints for the specified entities or repositories in a similar way like Spring Data REST does. This PR only includes Hibernate, but it's easily extendable for Mongo.

Reply:
Ah ok sorry. I’d use a single doc then and see how that evolves.

Reply:
So I'll keep the packages `io.quarkus.panache.rest.hibernate.orm.*` and rename the module `hibernate-orm-panache-rest` to `panache-rest-hibernate-orm`. Then we can add `panache-rest-mongo` later.
@FroMage what do you think?

Reply:
OK fine, thanks.